server/subscriber/MediaRequestSubscriber.ts

```diff
@@ -313,8 +313,11 @@ export class MediaRequestSubscriber
                 entity.requestedBy.id + '-' + entity.requestedBy.displayName,
             });
             userTag = await radarr.createTag({
-              label:
-                entity.requestedBy.id + '-' + entity.requestedBy.displayName,
+              label: (
+                entity.requestedBy.id +
+                '-' +
+                entity.requestedBy.displayName
+              ).replace(/\s+/g, '-'),
```

Comment:
IMHO make more sense to replace only on the `displayName` (no space in `id` or `'-'`), but that's up to you.